clients/roscpp/src/libros/transport/transport_tcp.cpp

```diff
@@ -40,8 +40,10 @@
 #include <ros/assert.h>
 #include <sstream>
 #include <boost/bind.hpp>
+#include <boost/lexical_cast.hpp>
 #include <fcntl.h>
 #include <errno.h>
+
```

Comment:
Not that I disagree with the reason for the change, but just for the sake of minimising changes with upstream, maybe remove this empty line insertion?